### ISSUE TITLE
Content-Type should be application/x-www-form-urlencoded  when 'use form data' is checked

### ISSUE THREAD

**Issue by tylkomat** [esphen/RESTED#103]
Html forms are at most of the times sent using Content-Type `application/x-www-form-urlencoded`. `multipart/form-data` only rarely used, usually when large amounts of data or binary data is sent.

**Comment by esphen** [comment]
Hey,

Yeah, bodies are scheduled to be reworked in 2.1 with new presets. A urlencoded preset will be a part of that
